mentat/app.py

```diff
@@ -98,7 +102,9 @@ def loop(
     )
     conv = Conversation(config, cost_tracker, code_file_manager)
 
-    cprint("Type 'q' or use Ctrl-C to quit at any time.\n", color="cyan")
+    get_interface().interact(
+        "Type 'q' or use Ctrl-C to quit at any time.\n", color="cyan"
+    )
```

Comment:
we could also add a convenience function to `interface.py` for common interact calls, like ones that just use `content` and `color`. It'd just take those two args and internally call `get_interface`